src/main/flight/mixer.c

```diff
@@ -345,6 +344,59 @@ static void applyFlipOverAfterCrashModeToMotors(void)
         }
     }
 }
+#ifdef USE_RPM_LIMITER
+static float getAverageRPM(void) // Placeholder function
+{
+    float averageRPM = 0;
+    for (int i = 0; i < getMotorCount(); i++) {
+        averageRPM += getDshotTelemetry(i);
+    }
+    return (averageRPM / (getMotorCount() * motorConfig()->motorPoleCount)) * 20;
+}
+
+static bool isMotorSaturated(void) //Placeholder function
+{
+    for (int i = 0; i < getMotorCount(); i++) {
+        if (motor[i] >= motorConfig()->maxthrottle) {
+            return true;
+        }
+    }
+    return false;
+}
+
+static void applyRPMLimiter(void)
```

Comment:
```suggestion
static void applyRPMLimiter(mixerRuntime_t *mixer)
```
Apply RPM Limiter to what? You can expand the parameter list to give a tip to what the limiter gets applied to (= `mixerRuntime`). You also need to replace all appearances of `mixerRuntime.` by `mixer->` in the function body.